src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.types.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    IColumn,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import {
+    IADTTwin,
+    IPropertyInspectorAdapter,
+    Locale,
+    Theme
+} from '../../../../Models/Constants';
+
+export interface IAdvancedSearchResultDetailsListProps {
+    filteredTwinsResult: IADTTwin[];
+    additionalColumns: IColumn[];
```

Comment:
let's name this `additionalProperties` or `searchedProperties` and make it type `string[]`